### ISSUE TITLE
is it possible to make the draggable more "precise" 

### ISSUE THREAD

**Issue by carlo-13** [NickPiscitelli/Glider.js#190]
I was just wondering if is it possible to make the draggable option more precise. For example, if look at the "Perspective View" here https://nickpiscitelli.github.io/Glider.js/, you often end up in between two slides, which doesn't look or feel really nice.
It there an option to prevent that ?
Thanks

**Comment by yvesvanbroekhoven** [comment]
Hi @carlo-13,

I think you can achieve this behavior by setting the `scrollLock` option to `true`.

The code for the perspective example would look like this:

```
window.glides = new Glider(document.getElementById('glider-persp'), {
  slidesToShow: 5,
  slidesToScroll: 1,
  scrollLock: true,
  draggable: true,
  dots: '#dots-per',
  arrows: {
    prev: '#glider-prev-per',
    next: '#glider-next-per'
  }
});```

Does that solves your issue?

**Comment by clarencefoy** [comment]
I have that option set to 'true', but it still gets stuck between slides quite easily. The closer you get to the edge, the less pronounced the effect is. When you scroll about halfway onto a slide, it's quite likely to get stuck. Especially if you wait a bit as well.

**Comment by themrdanielvp** [comment]
Yes it would be cool to have an option to "lock" the dragging so that the user only can drag one slide or page at a time.
